kafka-ui-react-app/src/components/common/Button/Button.tsx

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+
+export enum ButtonType {
+  PRIMARY = 'PRIMARY',
+  SECONDARY = 'SECONDARY',
+}
+
+export enum ButtonSize {
+  S = 'S',
+  M = 'M',
+  L = 'L',
+}
+
+const buttonStyles = {
+  PRIMARY: {
+    backgroundColor: {
+      normal: '#4F4FFF',
+      hover: '#1717CF',
+      active: '#1414B8',
+    },
+    color: '#FFFFFF',
+    invertedColors: {
+      normal: '#4F4FFF',
+      hover: '#1717CF',
+      active: '#1414B8',
+    },
+  },
+  SECONDARY: {
+    backgroundColor: {
+      normal: '#F1F2F3',
+      hover: '#E3E6E8',
+      active: '#D5DADD',
+    },
+    color: '#171A1C',
+    invertedColors: {
+      normal: '#73848C',
+      hover: '#454F54',
+      active: '#171A1C',
+    },
```

Comment:
colours must be defined in theme. cc: @abelgibayev

Reply:
Should I add this in this branch?